src/components/codecSupportHelper.ts

```diff
@@ -216,9 +221,9 @@ export function getSupportedMP4AudioCodecs(): Array<string> {
  * @returns Supported HLS video codecs.
  */
 export function getSupportedHLSVideoCodecs(): Array<string> {
-    // Currently the server does not support fmp4 which is required
-    // by the HLS spec for streaming H.265 video.
-    return ['h264'];
+  // Currently the server does not support fmp4 which is required
+  // by the HLS spec for streaming H.265 video.
```

Comment:
is this still true?

Reply:
Don't know as I did the tests just to check that nothing broke while changing between promises to async/await syntax (which was all that was needed to fix the linting and modernize the code according to our other projects)

This is outside the scope of this PR and should be determined later imo.

Reply:
As far as I can see, this file does not have any changes relating to this code anymore